src/module.c

```diff
@@ -13254,8 +13249,20 @@ int RM_RegisterNumericConfig(RedisModuleCtx *ctx, const char *name, long long de
     return REDISMODULE_OK;
 }
 
+/* Applies all default configurations for the parameters the module registered.
+ * Only call this function if the module would like to make changes to the
+ * configuration values before the actual values are applied by RedisModule_LoadConfigs.
+ * Otherwise continue calling RedisModule_LoadConfigs, it should already set the default values if needed.
+ * This sets an ordering for the possible source of a configuration value:
+ * 1. config values - e.g from the config file or CONFIG during loadex
+ * 2. module values - e.g from the module arguments
+ * 3. default values - if no other value was set
```

Comment:
i think your text was too long and hard to read.
not sure if it's clear what "module arguments" are, or if we really have to mention them.
maybe something like this is better.
```suggestion
 * This makes it possible to distinguish between default values and user provided values and apply other changes between setting the defaults and the user values.```